### PR TITLE
2020-08-31 Blog post: Rust, Arduino and Embedded Development as a Beginner: Part 1

### PR DESCRIPTION
**Pull request by cecton**
Don't hesitate to correct it! (I'm not native)

### CODE REVIEW

File: content/2020-08-31-newsletter-26.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ TODO Add news related to embedded Rust that are not about new crates releases he
 
 - TODO(remove, this is an example) the Rust compiler has gained cross compilation support for the Xtensa architecture!
 
+- A new blog post (in 2 parts) by [@cecton] targeted to experienced developers but embedded development beginners: [Rust, Arduino and Embedded Development as a Beginner: Part 1]
```
Comment (eldruin) [inline content/2020-08-31-newsletter-26.md:47]:
```suggestion
- [@cecton] wrote a blog post targeted to experienced developers but embedded development beginners: [Rust, Arduino and Embedded Development as a Beginner: Part 1]
```

**Review by eldruin — COMMENTED** [review]
Looks good to me. Usually we write the entries with a "<somebody> did <something>" style. Hence my suggestion. But it would be fine either way.

File: content/2020-08-31-newsletter-26.md (hunk 1 above)
Comment (cecton) [reply content/2020-08-31-newsletter-26.md:47]: Perfect!

**Review by eldruin — APPROVED** [review]
Great, thank you!

bors r+

**Pull request merged**